ccd-definition/AuthorisationCaseEvent/systemUpdate.json

```diff
@@ -71,6 +71,12 @@
     "UserRole": "caseworker-civil-systemupdate",
     "CRUD": "CRU"
   },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "NOTIFY_RESPONDENT_SOLICITOR2_FOR_CASE_HANDED_OFFLINE",
+    "UserRole": "caseworker-civil-systemupdate",
```

Comment:
There seems to be no consistency between notification events. Would need to confirm with the business / raise as another ticket.
But `NOTIFY_RESPONDENT_SOLICITOR2_FOR_DEFENDANT_RESPONSE_CC` provides R access to Solicitors and Admins, however the `NOTIFY_RESPONDENT_SOLICITOR1_FOR_CASE_HANDED_OFFLINE`  only gives CRU access for camunda. This event is not seen in Case History by **any** users

These new case events for respondent 2 are following the same permissions as the respondent 1 counterparts. But I am questioning here if the respondent 1 permissions might be an issue - as not all users can see these emails being triggered. 

Perhaps as it is a CC Email this is by design